_posts/2017-06-13-interview-with-eddy-colloton.md

```diff
@@ -0,0 +1,69 @@
+
+
+**Where do you use MediaConch? Do you use MediaConch primarily for file validation, for local policy checking, for in-house quality control, for quality testing for vendor files?**
+
+Primarily policy checking, as a form of quality assurance. At LPB, most of our files were being created through automated processes. Our legacy material was digitized using production workflows, to take advantage of existing institutional knowledge. This was very helpful, because we could then repurpose equipment, signal paths, and software. But, using these well worn workflows also meant that occasionally files would be encoded incorrectly, aspect ratio being one of the most common errors. We would check files against a MediaConch policy as a way of quickly flagging such errors, without having to invest time watching and reviewing the file ourselves.
+
+At the Denver Art Museum, we plan to use MediaConch in a similar way. The videotapes in the museum’s collection will be digitized by a vendor. Pre-ingest, I plan to do tests on the files for quality assurance. After fixity checks, I will check to make sure our target encoding and file format was met by the vendor using MediaConch. I intend to use the [Carnegie Archive’s python script from their GitHub](https://www.google.com/url?q=https://github.com/CarnegieHall/quality-control/blob/master/mediaconch/mediaconch-xmlreport-summary.py&sa=D&ust=1497302411714000&usg=AFQjCNGNBDXo2efUjLy6ztfFzk-hz3bIgQ) to automate this process. Once I know that the files are encoded to spec, I will be creating QCTools reports and playing back the files for visual QC. I’ve been following the [American Archive of Public Broadcasting’s QC procedures](https://www.google.com/url?q=https://americanarchivepb.wordpress.com/2017/04/12/pbs-newshour-digitization-project-update-ingest-and-digital-preservation-workflows/&sa=D&ust=1497302411715000&usg=AFQjCNEbvjjJMWxL0v4GLuwhRRx9l5EsKw) with interest to see if there’s any tricks I can cop from their workflow.
+
+
+**At what point in the archival process do you use MediaConch?**
+
+
+Basically pre-ingest for both LPB and the DAM. When using MediaConch as a policy checker, my goal is to make sure we’re not bothering to ingest a file that is not encoded to spec.
```

Comment:
IMO it is unclear (spec of what? format or DAM?) as it is a policy check: "LPB/DAM spec"?